e2e-tests/tests/semi-sync/02-check-orchestrator.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 30
+commands:
+- script: |
+    set -o errexit
+    set -o xtrace
+
+    source ../../functions
+
+    sleep 15 # wait for MySQL nodes to be discovered
```

Comment:
@hors we have "readyness" check above. Q: is node should be ready if it is not added to Orchestrator?